prototypes/form-wizard/src/foia.js

```diff
@@ -93,6 +93,10 @@ FOIA = {
     $('.unified-form').find(':input').prop('disabled', true);
     $('#submit').disabled = true;
     $('#confirmation').show();
+  },
+  toggleHelp: function (ev) {
+    var $help = $(ev.currentTarget).find('.tooltiptext');
+    $help.slideToggle();
```

Comment:
I think this will shift elements down and is generally a pattern I would avoid for help text (and use an absolute tooltip instead) but I'd consider it an optimization at this point so let's go with this and see if anyone has significant issues with it.